### ISSUE TITLE
Find bar not following accent color

### ISSUE THREAD

**Issue by Keith94** [spikespaz/firefox-nativedark#26]
I chose this accent color in Windows,

![applicationframehost_2018-05-15_21-29-13](https://user-images.githubusercontent.com/5490615/40096746-0d421356-5887-11e8-8b5d-f7fa619211e3.png)

And my findbar at the bottom looks like this as a result (is there a way to follow accent color?)

![firefox_2018-05-15_21-24-02](https://user-images.githubusercontent.com/5490615/40096752-1d1fe50a-5887-11e8-9a52-0347c7821f27.png)

**Comment by HRussell63250** [comment]
![image](https://user-images.githubusercontent.com/22746105/40266519-9a1dc580-5b44-11e8-9d4e-d60066edacbf.png)
in addition, if the difference mode is set to lighter, the findbar is unreadable.

**Comment by Keith94** [comment]
Yes, that's a pretty nasty bug!

**Comment by Keith94** [comment]
Fun fact: If `Unfocused theme` is set to `Light titlebar`, findbar is displayed correctly only when inactive.

![image](https://user-images.githubusercontent.com/5490615/40271792-6d706c0a-5b57-11e8-9142-a05def236a6a.png)

**Comment by HRussell63250** [comment]
Yup. I think it is related to the new theme API introduced:
https://developer.mozilla.org/en-US/Firefox/Releases/60

**Comment by martinguti95** [comment]
Hi,

I'm assigning this to me. I've been able to replicate what @Keith94 raised about the `Unfocused theme` and the `Light Titlebar` and I think that there's another style overriding the value of the accent color when focused

**Comment by martinguti95** [comment]
Checking the release notes of FF 61 https://developer.mozilla.org/en-US/docs/Mozilla/Firefox/Releases/61#WebExtensions , now the toolbar attributes apply also to the find bar. 

When the window is selected, a default value is applied to the `toolbar`. That's what is causing the bug.

**Comment by spikespaz** [comment]
@martinguti95 Yes, and that snippet of code in one of the commits solves the issue, however there are problems with it as you will find out. If you see another way to fix this, I'm all ears.
